installation/sql/sqlazure/joomla.sql

```diff
@@ -2579,6 +2579,8 @@ CREATE TABLE "#__template_styles" (
   "client_id" tinyint NOT NULL DEFAULT 0,
   "home" nvarchar(7) NOT NULL DEFAULT '0',
   "title" nvarchar(255) NOT NULL DEFAULT '',
+  "inheritable" smallint NOT NULL DEFAULT '0',
```

Comment:
```suggestion
  "inheritable" smallint NOT NULL DEFAULT 0,
```
@zero-24 Was this your test for me if I'm really awake?